design.tex

```diff
@@ -158,64 +156,62 @@ \subsubsection{Clearinghouse}\label{sec-ch}
 registration form \yanyan{cite url or show screenshot} at the 
 clearinghouse. The clearinghouse website shows 
 a list of available sensors, and each of their required accuracy 
-and access frequency. Aftering filling out this form, the researcher 
+and access frequency.\lois(does it show the "sensors" apart from the devices?) After filling out this form, the researcher 
```

Comment:
You say it "shows a s list of available sensors" as opposed to a "list of
available devices." So I wanted to know if the clearinghouse literally
offers experimenters "200 GPS" as opposed to 200 phones that have that
capability. It's not a huge point, but it does slightly change the
description if what the experimenter is choosing from...

On Wed, Nov 11, 2015 at 5:31 PM, Yanyan Zhuang <notifications@github.com>
wrote:

> In design.tex
> <https://github.com/yyzhuang/mobisys16-sensibility/pull/2#discussion_r44596773>
> :
>
> > @@ -158,64 +156,62 @@ \subsubsection{Clearinghouse}\label{sec-ch}
> >  registration form \yanyan{cite url or show screenshot} at the
> >  clearinghouse. The clearinghouse website shows
> >  a list of available sensors, and each of their required accuracy
> > -and access frequency. Aftering filling out this form, the researcher
> > +and access frequency.\lois(does it show the "sensors" apart from the devices?) After filling out this form, the researcher
>
> What does this mean?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yyzhuang/mobisys16-sensibility/pull/2/files#r44596773>
> .
>